taxi_service/settings.py

```diff
@@ -24,7 +24,7 @@
              "-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
DEBUG should not be hardcoded in settings.py. It's better to use environment variables for this setting as well. This will allow you to change the DEBUG mode depending on the environment (development, production, etc.) without changing the code.